voting-procedures.md

```diff
@@ -0,0 +1,268 @@
+  `@hiero-ledger/lf-staff @hiero-ledger/tsc @hiero-ledger/github-maintainers`
+- **Core project repos (hiero, hiero-website, tsc):** Use `/vote profile=hiero-project-maintainers-vote` → Notify
+  `@hiero-ledger/tsc @hiero-ledger/github-maintainers`
+- **SDK changes:** Use `/vote profile=hiero-sdk-<language>-maintainers-vote` → Notify
+  `@hiero-ledger/hiero-sdk-<language>-maintainers`
+- **Consensus Node changes:** Use `/vote profile=hiero-consensus-node-maintainers-vote` → Notify
+  `@hiero-ledger/hiero-consensus-node-maintainers`
+
+**Quick Actions:**
+
+- **Check vote status:** Comment `/check-vote`
+- **Cancel ongoing vote:** Comment `/cancel-vote`
+
+## What requires a vote
+
+A vote of Hiero maintainers is required in, but not necessarily limited to, the cases listed below:
+
+1. Changes to the membership of a project group (limited to the maintainers of the affected project)
+2. Changes to the permissions of a project group (limited to the maintainers of the affected project)
+3. Adding or removing a project group
+4. Changes to a project's exceptions from the guidelines and procedures defined in the governance repository.
+
+The intent is for maintainers to vote on any change to project membership, roles, and exceptions.
+
+## Choosing the Right Voting Profile
+
+Before initiating a vote, you need to select the appropriate voting profile based on what your change affects:
+
+| **Your Change Affects**                        | **Use Profile**                         | **Notify Teams**                                                            |
+|------------------------------------------------|-----------------------------------------|-----------------------------------------------------------------------------|
+| Governance repository itself                   | `default`                               | `@hiero-ledger/lf-staff @hiero-ledger/tsc @hiero-ledger/github-maintainers` |
+| Core project repos (hiero, hiero-website, tsc) | `hiero-project-maintainers-vote`        | `@hiero-ledger/tsc @hiero-ledger/github-maintainers`                        |
+| Consensus Node project                         | `hiero-consensus-node-maintainers-vote` | `@hiero-ledger/hiero-consensus-node-maintainers`                            |
+| Mirror Node project                            | `hiero-mirror-node-maintainers-vote`    | `@hiero-ledger/hiero-mirror-node-maintainers`                               |
+| Java SDK project                               | `hiero-sdk-java-maintainers-vote`       | `@hiero-ledger/hiero-sdk-java-maintainers`                                  |
+| Python SDK project                             | `hiero-sdk-python-maintainers-vote`     | `@hiero-ledger/hiero-sdk-python-maintainers`                                |
+| Other SDK projects                             | `hiero-sdk-<language>-maintainers-vote` | `@hiero-ledger/hiero-sdk-<language>-maintainers`                            |
```

Comment:
This table is not complete and will be incomplete whenever new projects are added. Instead of having this concrete table I suggest to only give 2 sample and make the naming pattern clear.